flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/FlinkConfigOptions.java

```diff
@@ -94,14 +94,17 @@ private FlinkConfigOptions() {}
   public static final ConfigOption<SplitAssignerType> TABLE_EXEC_SPLIT_ASSIGNER_TYPE =
       ConfigOptions.key("table.exec.iceberg.split-assigner-type")
           .enumType(SplitAssignerType.class)
-          .defaultValue(SplitAssignerType.SIMPLE)
+          .noDefaultValue()
```

Comment:
We cannot use the default value. When the user does not configure it, we need it to be null, so that we can further determine whether locality inference is supported by checking the I/O type